README.md

```diff
@@ -1,24 +1,238 @@
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* Database creation
+<!-- PROJECT DESCRIPTION -->
 
-* Database initialization
+# 📖 GymFit <a name="A web app  that helps make reservations for a fitness activity, eg: Yoga at the GymFit center"></a>
 
-* How to run the test suite
+- **GymFit** is a web application that allows users to view and reserve a fitness activity, i.e. Yoga class, Gym e.t.c. A user can reserve multiple fitness activities and a fitness activity can have multiple reservations on separate dates.
+- The application features user based authorization, a basic user can read and manage their user specific reservations while the admin can manage all operations in the application. I.e. create and delete fitness_activities e.t.c
 
-* Services (job queues, cache servers, search engines, etc.)
+![final-capstone-uml](https://user-images.githubusercontent.com/41898352/206205814-1bc35f56-3c1b-4c17-b3c9-895b374860ef.png)
 
-* Deployment instructions
 
-* ...
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- **Authenticated endpoints wit jwt**
+- **User Authorization**
+- **User Authentication with devise**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link will be available soon](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+ 
+- Ruby runtime environment
+- Postgresql
+- Rails
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+- Clone the repository using `git@github.com:XsidX/final-capstone-back-end.git`
+- Navigate to the folder using `cd final-capstone-back-end`
+- Use the following commands to
+    - install necessary gems `bundle install`
+    - create database `rails db:create`
+    - run migrations and create default user as admin `rails db:migrate db:seed` 
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+ rails s -p 3001
+```
+
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+Rspec spec
+```
```

Comment:
- [ ] The correct command here is `rspec spec`, not `Rspec spec`.  Kindly remember that these commands are case-sensitive.  👍
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍